packages/neuron-wallet/src/services/rpc-service.ts

```diff
@@ -29,6 +30,23 @@ export default class RpcService {
     return BlockHeader.fromSDK(result)
   }
 
+  public async getTransactionIncludeRejected(
```

Comment:
There exists a comment on the `getTransaction` function.

```
  /**
   * TODO: rejected tx should be handled
   * {
   *   transaction: null,
   *   txStatus: { blockHash: null, status: 'rejected' }
   * }
   */

How about using `getTransaction` to handle the `reject` condition and remove the comment?
```

Reply:
> There exists a comment on the `getTransaction` function.
> 
> ```
>   /**
>    * TODO: rejected tx should be handled
>    * {
>    *   transaction: null,
>    *   txStatus: { blockHash: null, status: 'rejected' }
>    * }
>    */
> 
> How about using `getTransaction` to handle the `reject` condition and remove the comment?
> ```

Considering the normal call to the getTransaction method, there is no need to separately determine whether the transaction is rejected or not, only whether the return value is undefined or not
So the getTransactionIncludeRejected method is used separately to support the case where rejected transactions need to be handled.

Reply:
If the caller does not need the reject status, it can ignore that return result. And check whether the transaction is null or the result is undefined.